tests/test_inclined_exponential.py

```diff
@@ -30,155 +38,319 @@
     sys.path.append(os.path.abspath(os.path.join(path, "..")))
     import galsim
 
+# Save images used in regression testing for manual inspection?
+save_profiles = False
+
 # set up any necessary info for tests
 # Note that changes here should match changes to test image files
 image_dir = './inclined_exponential_images'
 
 # Values here are strings, so the filenames will be sure to work (without truncating zeros)
-fluxes = ("1.0", "10.0", "0.1", "1.0", "1.0", "1.0")
-image_inc_angles = ("0.0", "1.3", "0.2", "0.01", "0.1", "0.78")
-image_scale_radii = ("3.0", "3.0", "3.0", "3.0", "2.0", "2.0")
-image_scale_heights = ("0.3", "0.5", "0.5", "0.5", "1.0", "0.5")
-image_pos_angles = ("0.0", "0.0", "0.0", "0.0", "-0.2", "-0.2")
+
+# Tuples are in the order: (flux, n, inclination angle (radians), scale radius, scale height,
+#                           truncation factor, position angle)
+
+# Parameter sets valid for the inclined exponential case
+inclined_exponential_test_parameters = (
+    ("1.0", "1.0", "0.0" , "3.0", "0.3", "0.0", "0.0"),
+    ("10.0", "1.0", "1.3" , "3.0", "0.5", "0.0", "0.0"),
+    ("0.1", "1.0", "0.2" , "3.0", "0.5", "0.0", "0.0"),
+    ("1.0", "1.0", "0.01", "3.0", "0.5", "0.0", "0.0"),
+    ("1.0", "1.0", "0.1" , "2.0", "1.0", "0.0", "-0.2"),
+    ("1.0", "1.0", "0.78", "2.0", "0.5", "0.0", "-0.2"),)
+
+# Parameter sets valid only for the inclined Sersic case
+inclined_sersic_test_parameters = (
+    ("2.0", "1.5", "0.0" , "3.0", "0.3", "0.0", "0.0"),
+    ("1.5", "2.0", "0.0" , "3.0", "0.3", "0.0", "0.0"),
+    ("1.1", "0.5", "0.0" , "3.0", "0.3", "0.0", "0.0"),
+    ("10.0", "1.0", "0.1" , "1.9", "0.3", "4.5", "7.3"),
+    ("1.0e6", "1.5", "0.2" , "2.1", "0.2", "3.9", "-0.9"),
+    ("1.0e-6", "2.0", "0.3" , "3.4", "0.1", "5.0", "0.6"),
+    ("2.3e4", "0.5", "0.5" , "1.8", "0.5", "2.5", "0.3"),)
```

Comment:
You have special code for cosi < 0.01 and cosi > 0.95.  It doesn't look like you're testing that with these inclination values (other than the cosi = 1 case).  Should add cases that make sure to hit those branches.